### ISSUE TITLE
Add warning or confirmation on an empty folder initial sync

### ISSUE THREAD

**Issue by sicenul** [MasterSoft24/CloudCross#87]
There is a "Please Note" statement on usage instruction on the web, but I think this is not enough.
It should show a warning or a confirmation when initiate sync from an empty folder.
I accidentally delete all remote files on my google drive, since I missed that "please note" statement.
I know I should read all the usage instruction :(

**Comment by MasterSoft24** [comment]
Hi, I'm so sorry that you lost your files but all files when deleting must be placed in a Trash bin on GoogleDrive. Please check this. About "please note" I'm thinking you right and this is useful

**Comment by LukeGarou77** [comment]
Yes, this was extremely confusing-- I still haven't seen the "Please note", and was very relieved to find my files in Drive trash. I was gutted for a second there. Of course with all that's going on, having to bypass the "safety warnings" due to Russian certificates not being recognized by GAE/NATO allies, I was worried someone had really screwed me. 
Would certainly like to see something done about this, and a more explicit warning.
